xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -89,16 +91,21 @@
 
   private static final SchemaFactory XSD_SCHEMA_FACTORY = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
 
+  private static final URL CORE_SCHEMA_URL = XmlConfiguration.class.getResource("/ehcache-core.xsd");
+  private static final String CORE_SCHEMA_NAMESPACE = "http://www.ehcache.org/v3";
+  private static final String CORE_SCHEMA_ROOT_ELEMENT = "config";
+  private static final String CORE_SCHEMA_JAXB_MODEL_PACKAGE = "org.ehcache.xml.model";
```

Comment:
Oh, I know... I didn't like the way Alex did it before... now that someone is touching this I can get the 'fixed'!